packages/calcite-components/src/components/dialog/dialog.tsx

```diff
@@ -366,6 +372,19 @@ export class Dialog
   // #endregion
 
   // #region Private Methods
+
+  private handleFocusTrapDisabled(): void {
+    if (!this.open) {
+      deactivateFocusTrap(this);
```

Comment:
can just return here and not deactivate. onClose should handle deactivating.

Reply:
good idea